_studio/shared/src/mfx_scd.cpp

```diff
@@ -11816,12 +11816,13 @@ mfxStatus SceneChangeDetector::SubSampleImage(mfxU32 srcWidth, mfxU32 srcHeight,
     if (NULL == m_pCmProgram)
     {
         if(m_platform == PLATFORM_INTEL_HSW)
-         res = m_pCmDevice->LoadProgram((void *)asc_genx_hsw, sizeof(asc_genx_hsw), m_pCmProgram);
-        else if (m_platform == PLATFORM_INTEL_SKL)
-         res = m_pCmDevice->LoadProgram((void *)asc_genx_skl, sizeof(asc_genx_skl), m_pCmProgram);
-     else
-         res = m_pCmDevice->LoadProgram((void *)asc_genx_bdw, sizeof(asc_genx_bdw), m_pCmProgram);
-     if(res != 0 ) return MFX_ERR_DEVICE_FAILED;
+            res = m_pCmDevice->LoadProgram((void *)asc_genx_hsw, sizeof(asc_genx_hsw), m_pCmProgram);
+        else if (m_platform == PLATFORM_INTEL_SKL || m_platform == PLATFORM_INTEL_CFL ||
+                 m_platform == PLATFORM_INTEL_CNL || m_platform == PLATFORM_INTEL_ICL)
```

Comment:
1. CNL/ICL should have their own kernels
2. please don't add ICL here now

Reply:
Removed CNL/ICL